### ISSUE TITLE
Error on 3.1.2 container startup

### ISSUE THREAD

**Issue by eptw4by** [bridgecrewio/checkov-action#155]
After bumping up [version to 3.1.2](https://github.com/bridgecrewio/checkov-action/commit/2ed3219762045085b5a7e479c8e84246bfe01ba9), we are observing following errors in our executions:
```
Error: -20 14:30:40,227 [MainThread  ] [ERROR]  Exception traceback:
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
                    ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/multiprocessing/pool.py", line 51, in starmapstar
    return list(itertools.starmap(args[0], args[1]))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/common/runners/runner_registry.py", line 788, in _parallel_run
    report = runner.run(
             ^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/sast/runner.py", line 53, in run
    bc_integration.set_s3_client()
  File "/usr/local/lib/python3.11/site-packages/checkov/common/bridgecrew/platform_integration.py", line 397, in set_s3_client
    raise ValueError("Credentials for client are not set")
ValueError: Credentials for client are not set
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/checkov/main.py", line 523, in run
    self.scan_reports = runner_registry.run(
                        ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/common/runners/runner_registry.py", line 158, in run
    parallel_runner_results = parallel_runner.run_function(
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/common/parallelizer/parallel_runner.py", line 52, in run_function
    return self._run_function_multiprocess_spawn(func, items, group_size)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/common/parallelizer/parallel_runner.py", line 122, in _run_function_multiprocess_spawn
    return p.starmap(func, items, chunksize=group_size)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/multiprocessing/pool.py", line [37](https://github.com/mckesson/b2b-connect-ordering-presentation-data-integration-iac/actions/runs/6931519354/job/18853413656#step:4:38)5, in starmap
    return self._map_async(func, iterable, starmapstar, chunksize).get()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/multiprocessing/pool.py", line 774, in get
    raise self._value
ValueError: Credentials for client are not set
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
                    ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/multiprocessing/pool.py", line 51, in starmapstar
    return list(itertools.starmap(args[0], args[1]))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/common/runners/runner_registry.py", line 788, in _parallel_run
    report = runner.run(
             ^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/sast/runner.py", line 53, in run
    bc_integration.set_s3_client()
  File "/usr/local/lib/python3.11/site-packages/checkov/common/bridgecrew/platform_integration.py", line [39](https://github.com/mckesson/b2b-connect-ordering-presentation-data-integration-iac/actions/runs/6931519354/job/18853413656#step:4:40)7, in set_s3_client
    raise ValueError("Credentials for client are not set")
ValueError: Credentials for client are not set
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/bin/checkov", line 9, in <module>
    sys.exit(Checkov().run())
             ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/main.py", line [52](https://github.com/mckesson/b2b-connect-ordering-presentation-data-integration-iac/actions/runs/6931519354/job/18853413656#step:4:53)3, in run
    self.scan_reports = runner_registry.run(
                        ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/common/runners/runner_registry.py", line 1[58](https://github.com/mckesson/b2b-connect-ordering-presentation-data-integration-iac/actions/runs/6931519354/job/18853413656#step:4:59), in run
    parallel_runner_results = parallel_runner.run_function(
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/common/parallelizer/parallel_runner.py", line 52, in run_function
    return self._run_function_multiprocess_spawn(func, items, group_size)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/common/parallelizer/parallel_runner.py", line 122, in _run_function_multiprocess_spawn
    return p.starmap(func, items, chunksize=group_size)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/multiprocessing/pool.py", line 375, in starmap
    return self._map_async(func, iterable, starmapstar, chunksize).get()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/multiprocessing/pool.py", line 7[74](https://github.com/mckesson/b2b-connect-ordering-presentation-data-integration-iac/actions/runs/6931519354/job/18853413656#step:4:75), in get
    raise self._value
ValueError: Credentials for client are not set
```

This is happening through multiple repositories

**Comment by maximeancellin** [comment]
Hi, 

I have same issue on checkov version: `3.1.1` and action: `bridgecrewio/checkov-action@v12`

On friday when I ran my check all my pipelines ended by success.
Today on same resources, all ended on following error:

```
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
                    ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/multiprocessing/pool.py", line 51, in starmapstar
    return list(itertools.starmap(args[0], args[1]))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/common/runners/runner_registry.py", line 788, in _parallel_run
    report = runner.run(
             ^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/secrets/runner.py", line 131, in run
    bc_integration.set_s3_client()
  File "/usr/local/lib/python3.11/site-packages/checkov/common/bridgecrew/platform_integration.py", line 397, in set_s3_client
    raise ValueError("Credentials for client are not set")
ValueError: Credentials for client are not set
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/checkov/main.py", line 523, in run
    self.scan_reports = runner_registry.run(
                        ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/common/runners/runner_registry.py", line 158, in run
    parallel_runner_results = parallel_runner.run_function(
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/common/parallelizer/parallel_runner.py", line 52, in run_function
    return self._run_function_multiprocess_spawn(func, items, group_size)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/common/parallelizer/parallel_runner.py", line 122, in _run_function_multiprocess_spawn
    return p.starmap(func, items, chunksize=group_size)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/multiprocessing/pool.py", line [37](https://github.com/trusk-official/trusk-infra-iac/actions/runs/6930867496/job/18851358557#step:4:38)5, in starmap
    return self._map_async(func, iterable, starmapstar, chunksize).get()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/multiprocessing/pool.py", line 774, in get
    raise self._value
ValueError: Credentials for client are not set
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
                    ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/multiprocessing/pool.py", line 51, in starmapstar
    return list(itertools.starmap(args[0], args[1]))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/common/runners/runner_registry.py", line 788, in _parallel_run
    report = runner.run(
             ^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/secrets/runner.py", line 131, in run
    bc_integration.set_s3_client()
  File "/usr/local/lib/python3.11/site-packages/checkov/common/bridgecrew/platform_integration.py", line [39](https://github.com/trusk-official/trusk-infra-iac/actions/runs/6930867496/job/18851358557#step:4:40)7, in set_s3_client
    raise ValueError("Credentials for client are not set")
ValueError: Credentials for client are not set
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/bin/checkov", line 9, in <module>
    sys.exit(Checkov().run())
             ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/main.py", line [52](https://github.com/trusk-official/trusk-infra-iac/actions/runs/6930867496/job/18851358557#step:4:53)3, in run
    self.scan_reports = runner_registry.run(
                        ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/common/runners/runner_registry.py", line 158, in run
    parallel_runner_results = parallel_runner.run_function(
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/common/parallelizer/parallel_runner.py", line 52, in run_function
    return self._run_function_multiprocess_spawn(func, items, group_size)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/checkov/common/parallelizer/parallel_runner.py", line 122, in _run_function_multiprocess_spawn
    return p.starmap(func, items, chunksize=group_size)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/multiprocessing/pool.py", line 375, in starmap
    return self._map_async(func, iterable, starmapstar, chunksize).get()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/multiprocessing/pool.py", line 774, in get
    raise self._value
ValueError: Credentials for client are not set
```

Thanks by advance for support.

**Comment by nvukas59** [comment]
We have also the same issue on 3.1.2, it's seems to be related to your commits @gruebel. Could you please tell us when this will be resolved ?

**Comment by nvukas59** [comment]
The issue has been resolved by [Bump checkov container version to 3.1.3](https://github.com/bridgecrewio/checkov-action/commit/c3db867e93c463275ef08a0b13926bb206f8467c) ! It's working fine now.
